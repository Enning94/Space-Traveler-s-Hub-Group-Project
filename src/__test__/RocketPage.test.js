import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import Rockets from '../Components/Pages/RocketsPage';
import store from '../Redux/configureStore';

describe('Renders correctly', () => {
  it('Renders page correctly', () => {
    const { container } = render(<Provider store={store}><Rockets /></Provider>);
    expect(container).toMatchSnapshot();
  });
});
