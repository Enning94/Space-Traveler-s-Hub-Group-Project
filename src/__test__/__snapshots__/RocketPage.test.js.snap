// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly Renders page correctly 1`] = `
<div>
  <main
    class="p-10"
  >
    <ul
      class="grid grid-cols-1 gap-4"
    />
  </main>
</div>
`;
